src/main/java/com/laytonsmith/abstraction/bukkit/BukkitMCCommand.java

```diff
@@ -209,26 +213,33 @@ public String toString() {
 
 	@Override
 	public List<String> handleTabComplete(MCCommandSender sender, String alias, String[] args) {
+		Environment env;
+		try {
+			env = Static.GenerateStandaloneEnvironment();
```

Comment:
I cannot imagine that this is right. Though, since we can't change the signature of this function, the solution may be pretty interesting as well.

Reply:
Can we not use the env from the tabcomplete closure below instead of creating a new env?

Reply:
Grab the env from the closure?